mlir/include/mlir/Dialect/SPIRV/IR/SPIRVIntelExtOps.td

```diff
@@ -131,7 +131,7 @@ class SPIRV_IntelSplitBarrierOp<string mnemonic>
   let results = (outs);
 
   let assemblyFormat = [{
-    $execution_scope `,` $memory_scope `,` $memory_semantics attr-dict
+    $execution_scope $memory_scope $memory_semantics attr-dict
```

Comment:
@kuhar, apparently I didn't push this in the PR I merged before, but in this branch...

Reply:
Done